pkg/monitor/monitor.go

```diff
@@ -59,66 +59,68 @@ func NewAgentMonitor(client clientset.Interface, ovsBridge string, nodeName stri
 // Then updates AntreaControllerInfo CRD every 60 seconds if there is any change.
 func (monitor *controllerMonitor) Run(stopCh <-chan struct{}) {
 	klog.Info("Starting Antrea Controller Monitor")
-	controllerCRD := monitor.getControllerCRD()
-	var err error = nil
+
+	crdName := "antrea-controller"
+	controllerCRD, err := monitor.createControllerCRD(crdName)
+	if err != nil {
+		klog.Errorf("Failed to create controller monitoring CRD %v : %v", controllerCRD, err)
+		return
+	}
+
+	defer monitor.deleteControllerCRD(crdName)
+
+	ticker := time.NewTicker(60 * time.Second)
 	for {
```

Comment:
[This comment](https://github.com/vmware-tanzu/antrea/pull/159#discussion_r351948051) may help.
> Please use one of the utility functions in https://godoc.org/k8s.io/apimachinery/pkg/util/wait for this. It will be more consistent with the rest of the test code and easier to read.

Reply:
Good point, have updated this part.